### ISSUE TITLE
Build iOS application: failed with code: 65 after declaring the datastore dependency in version catalog

### ISSUE THREAD

**Issue by phuocth** [russhwolf/multiplatform-settings#159]
After declaring the datastore setting in the libs.versions.toml file: 
`multiplatform-settings-datastore = { module = "com.russhwolf:multiplatform-settings-datastore", version.ref = "multiplatformSettings" }`

`multiplatformSettings = "1.0.0"`

The build task for iOS application failed:
```> Task :build-logic:convention:inspectClassesForKotlinIC

FAILURE: Build failed with an exception.
* What went wrong:
An exception occurred applying plugin request [id: android.app']
Could not find implementation class 'buildlogic.AndroidAppConventionPlugin' for plugin 'android.app' specified in jar:file:/Users/masked/.gradle/caches/jars-9/o_ccf471e62a77ae1f23ad05847afd4305/convention.jar!/META-INF/gradle-plugins/android.app.properties.
```
But when I removed the declaration, the build went through.

**Comment by russhwolf** [comment]
It's really hard to guess what's happening here without knowing more about your project

**Comment by phuocth** [comment]
Yeah thanks @russhwolf . I was not really sure what was going on. Since when I got the build error and after trying many times to restart and clear the android studio cache, there was still no help. After the I've just tried commenting out the above dependency declaration `:multiplatform-settings-datastore` and added the plain implementation in androidMain as `implementation("com.russhwolf:multiplatform-settings-datastore:1.0.0")`, then it went through. 
However, currently when I add again the datastore dependency into the `toml` file, the build still went through as normally. 
So, I think of myself it was something weird with my android studio caching. I would like to close this bug since it's not reproducible for me anymore.
